Docker/build/Wikibase/Dockerfile

```diff
@@ -4,10 +4,7 @@ ARG MEDIAWIKI_IMAGE_NAME
 ARG MEDIAWIKI_IMAGE_VERSION
 FROM ubuntu:xenial as unpacker
 
-RUN apt-get update && \
```

Comment:
we aren't curling, unzipping or jq'ing so this can go. However this could probably just be done in the actual container it's moved to directly.